src/app/(customers)/(components)/CustomerContactsSection.tsx

```diff
@@ -0,0 +1,46 @@
+'use client';
+
+import AddContactForm from '@/app/(contacts)/(components)/(forms)/AddContactForm';
+import { ContactTableColumns } from '@/app/(contacts)/(components)/(tables)/ContactTableColumns';
+import { Button } from '@/components/ui/button';
+import { DataTable } from '@/components/ui/data-table';
+import { useState } from 'react';
+import { CustomerWithRelations } from '../types';
+
+type Props = {
+  customer: CustomerWithRelations;
+};
+
+const CustomerContactsSection = (props: Props) => {
+  const [open, setOpen] = useState(false);
```

Comment:
widzę, że w kolejnym miejscu używać takiego toggle. Może warto wynieść to to reużywalnego hooka?